assets/helpers/switch.js

```diff
@@ -9,27 +9,3 @@ type SwitchObject = {
 export type Switches = {
   [string]: SwitchObject,
 };
-
-
-// ----- Default state ----- //
-
-const defaultSwitches: Switches = {
-  oneOffPaymentMethods: {
-    stripe: 'ON',
-
-    // === 24-7 ===
```

Comment:
We should add a similar comment in the new switch file, for 24/7 guidance. cc: @johnduffell

Reply:
@johnduffell it will live on forever in our memories